### PR TITLE
Removed Preview note; changed Add res. to See also

### PR DESCRIPTION
**Pull request by Linda-Editor**
Removed Preview note per LindaLu. Also changed Additional resources to See also per our new guidelines.

### CODE REVIEW

File: reference/manifest/createaddincommands.md [hunk 1]
```diff
@@ -27,7 +29,7 @@ This article describes how to edit your manifest to define add-in commands. The
     
 ## Step 1: Start from a sample
 
-We strongly recommend that you start from one of the samples we provide in  [Office Add-in Commands Samples](https://github.com/OfficeDev/Office-Add-in-Command-Sample). Optionally, you can create your own manifest by following the steps in this guide. You can validate your manifest using the XSD file in the Office Add-in Commands Samples site. Ensure that you have read  [Add-in commands for Excel, Word and PowerPoint (Preview)](http://msdn.microsoft.com/library/5c898275-e47f-4e97-97df-098b769620d6%28Office.15%29.aspx) before using add-in commands.
+We strongly recommend that you start from one of the samples we provide in  [Office Add-in Commands Samples](https://github.com/OfficeDev/Office-Add-in-Command-Sample). Optionally, you can create your own manifest by following the steps in this guide. You can validate your manifest using the XSD file in the Office Add-in Commands Samples site. Ensure that you have read  [Add-in commands for Excel, Word and PowerPoint](http://msdn.microsoft.com/library/5c898275-e47f-4e97-97df-098b769620d6%28Office.15%29.aspx) before using add-in commands.
```
Comment (ElizabethSamuel-MSFT) [inline reference/manifest/createaddincommands.md]: Can you update this msdn link to https://docs.microsoft.com/office/dev/add-ins/design/add-in-commands?

Comment (ElizabethSamuel-MSFT) [reply reference/manifest/createaddincommands.md]: There are several instances - could you update all of them?

Comment (ElizabethSamuel-MSFT) [reply reference/manifest/createaddincommands.md]: Also, there should be a comma after Word i.e. Add-in commands for Excel, Word, and PowerPoint

Comment (Linda-Editor) [reply reference/manifest/createaddincommands.md]: @ElizabethSamuel-MSFT  Made changes at https://github.com/OfficeDev/office-js-docs/pull/1452. Closing this PR.

**Comment by Linda-Editor** [comment]
Closing. Pick up work again post-migration after August 30.

**Comment by ElizabethSamuel-MSFT** [comment]
https://github.com/OfficeDev/office-js-docs/pull/1453 - removed createaddincommands.md

**Comment by ElizabethSamuel-MSFT** [comment]
note for auth.getAccessTokenAsync added in PR https://github.com/OfficeDev/office-js-docs-reference/pull/72/files#diff-33c99adf54dd7a75c8b8209d785ec890

**Pull request merged**